src/ui/views/ViewUSB/Index.vue

```diff
@@ -38,11 +36,13 @@ export default {
   components: {Goggles},
   data() {
     return {
+      compatibleBrowser: false,
       pairedGoggles: [],
       showHelp: false,
     }
   },
-  created() {
+  mounted() {
+    this.compatibleBrowser = navigator.userAgent.indexOf('Chrome') !== -1;
```

Comment:
Can you make this computed, rather than a data property. You shouldn't be able to edit `compatibleBrowser`, so it makes sense to put it as a computed

Reply:
You're right, I just updated the PR. 👍🏼